README.rst

```diff
@@ -37,9 +37,17 @@ Current information can always be found at the NIPY website::
 Mailing Lists
 =============
 
-Please see the developer's list::
+For questions on how to use nipy, please see NeuroStars:
```

Comment:
I don't actually use neurostars.  I'm not sure about Alexis or Bertrand, but I can't see them mentioned on neurostars at a quick search.  I personally don't like the idea of splitting user and developer questions.  So, I think we should continue to recommend the (new) mailing list for user questions.